packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const sql = `    
+    SET statement_timeout = 0;
+    SET lock_timeout = 0;
+    SET idle_in_transaction_session_timeout = 0;
+    SET client_encoding = 'UTF8';
+    SET standard_conforming_strings = on;
+    SELECT pg_catalog.set_config('search_path', '', false);
+    SET check_function_bodies = false;
+    SET xmloption = content;
+    SET client_min_messages = warning;
+    SET row_security = off;
+    
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
+    
+    COMMENT ON EXTENSION "uuid-ossp" IS 'generate universally unique identifiers (UUIDs)';
+    
+    CREATE TYPE public.application_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    
+    ALTER TYPE public.application_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.application_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.application_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.environment_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.environment_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.environment_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.environment_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_csp_enum AS ENUM (
+        'CSP A',
+        'CSP B'
+    );
+    
+    ALTER TYPE public.portfolio_csp_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.task_order_filescanstatus_enum AS ENUM (
+        'pending',
+        'accepted',
+        'rejected'
+    );
+    
+    ALTER TYPE public.task_order_filescanstatus_enum OWNER TO postgres;
+    
+    SET default_tablespace = '';
+    SET default_table_access_method = heap;
```

Comment:
I *think* we can omit these because they're defaults. https://www.postgresql.org/docs/13/runtime-config-client.html

Reply:
Removed `default_tablespace` in a8639253ecd206acabff6bacc6d11b5e8939a0bc

Reply:
Per linked docs, _heap_ is the default value for `default_table_access_method`.  Will remove.
> This parameter specifies the default table access method to use when creating tables or materialized views if the CREATE command does not explicitly specify an access method, or when SELECT ... INTO is used, which does not allow to specify a table access method. The default is heap.

Reply:
Removed `default_table_access_method` in 75274233e973428925996f95b810cfffecb3ccea